pgmpy/models/BayesianModel.py

```diff
@@ -442,6 +442,12 @@ def get_independencies(self, latex=False):
                     if independent_variables:
                         independencies.add_assertions([start, independent_variables,
                                                        observed])
+            independencies_list = independencies.get_assertions()
+            local_independencies_list = self.local_independencies(start)
+            assertions_list = local_independencies_list.get_assertions()
+            for assertion in assertions_list:
```

Comment:
@ankurankan I wanted to discuss this with you earlier.  
But thats fine. I will make the necessary changes.
Also, using the `set` implementation will give a performance boost.